packages/notebook/src/widget.ts

```diff
@@ -1213,6 +1224,7 @@ export class Notebook extends StaticNotebook {
         this.activeCellIndex = newActiveCellIndex;
       }
     }
+    this.addFooter();
```

Comment:
Could you add in the `constructor` because here it will be called every time a cell is added or removed.

Reply:
This has been changed.

Reply:
@fcollonval If I am not mistaking it seems that moving `addFooter`to the constructor causes the footer widget to become invisible or just not added after the last cell of the notebook. I don't know why.